### PR TITLE
tracker: update to 2.0.0

### PR DESCRIPTION
**Pull request by Sir-Boops**
* Only tested on x86_64

### CODE REVIEW

File: srcpkgs/tracker/template [hunk 1]
```diff
@@ -32,6 +32,10 @@ if [ -z "$CROSS_BUILD" ]; then
 	build_options_default="gir"
 fi
 
+case "$XBPS_TARGET_MACHINE" in
+	arm*) broken="sqlite will not cross compile with fts5 support"
+esac
```
Comment (Gottox) [inline srcpkgs/tracker/template]: sqlite has `--enable-fts5` enabled. Is this a noop?

Comment (Sir-Boops) [reply srcpkgs/tracker/template]: For cross it seems like it. As non cross compiled versions work fine

Comment (Gottox) [reply srcpkgs/tracker/template]: Strange. But ok. Do you have time to fix sqlite here? - I can give it a try tomorrow.

Comment (Sir-Boops) [reply srcpkgs/tracker/template]: I have some time today to work at it.

**Comment by ebfe** [comment]
This left the common/shlibs entries for libtracker-1.0. brasero/bijben need a revbump because they depend on the old sonames.

**Comment by Gottox** [comment]
Even worse: It's an API change, not an ABI change.

**Pull request merged**